modules/ROOT/pages/7.9.0-release-notes.adoc

```diff
@@ -157,7 +157,12 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 // #TINY-vwxyz1
 
 // CCFR here.
+=== If `style_formats` is empty the button is now disabled.
+// #TINY-12005
 
+Previously, when the xref:user-formatting-options.adoc#style_formats[style_formats] configuration was empty or undefined, the **Formats** toolbar button remained active, resulting in a dropdown menu that appeared empty and non-functional. This behavior caused confusion, as users were presented with an unresponsive interface element that suggested available formatting options when none existed.
```

Comment:
Undefined worked, that just went to our default value. It was specifically when giving an empty list. Sorry for being unclear.

Reply:
```suggestion
Previously, when the xref:user-formatting-options.adoc#style_formats[style_formats] configuration was explicitly set to an empty list, the **Formats** toolbar button remained enabled, displaying an empty and non-functional dropdown menu. This led to confusion, as the button appeared interactive and suggested available formatting options, despite none being configured.
```